pkg/schedule/schedule.go

```diff
@@ -38,6 +39,34 @@ func IsInScheduleRanges(now time.Time, timeRanges []api.DatetimeRange) (bool, er
 }
 
 func IsInScheduleRange(now time.Time, start string, end string) (bool, error) {
+	if len(conf.Config.Basic.TimeZone) > 0 {
+		loc, err := time.LoadLocation(conf.Config.Basic.TimeZone)
+		if err != nil {
+			return false, err
+		}
+		return isInTimeRangeWithLoc(now, start, end, loc)
+	}
+	return isInTimeRange(now, start, end)
+}
+
+func isInTimeRangeWithLoc(now time.Time, start string, end string, loc *time.Location) (bool, error) {
+	s, err := time.ParseInLocation(layout, start, loc)
```

Comment:
Can we use cast.InterfaceToTime? So that you don't need to handle timezone by yourself here. And it can support more time format. It is the universal time cast function in this project.